examples/camera-app/camera-common/include/camera-device-interface.h

```diff
@@ -263,6 +300,21 @@ class CameraDeviceInterface
         virtual uint8_t GetMicrophoneMaxLevel() = 0;
         virtual uint8_t GetMicrophoneMinLevel() = 0;
 
```

Comment:
Need the Setters/Getters for the ImageControl attributes.

Reply:
Done in the latest commit